--- x/qtbase/src/plugins/platforms/cocoa/qprintengine_mac_p.h
+++ y/qtbase/src/plugins/platforms/cocoa/qprintengine_mac_p.h
@@ -52,6 +52,7 @@
 //

 #include <QtCore/qglobal.h>
+#include <qpa/qplatformprintdevice.h>

 #ifndef QT_NO_PRINTER

--- old/qtbase/src/plugins/plugins.pro	2017-06-28 10:54:29.000000000 +0100
+++ new/qtbase/src/plugins/plugins.pro	2019-01-02 16:29:41.817231643 +0000
@@ -8,6 +8,3 @@
     qtConfig(imageformatplugin): SUBDIRS *= imageformats
     !android:qtConfig(library): SUBDIRS *= generic
 }
-
-!winrt:!wince:qtHaveModule(printsupport): \
-    SUBDIRS += printsupport
